core/app/service/setting.go

```diff
@@ -452,3 +513,17 @@ func checkCertValid() error {
 
 	return nil
 }
+
+func loadPort(address string) string {
+	re := regexp.MustCompile(`(?:(?:\[([0-9a-fA-F:]+)\])|([^:/\s]+))(?::(\d+))?$`)
+	matches := re.FindStringSubmatch(address)
+	if len(matches) <= 0 {
+		return ""
+	}
+	var port string
+	port = matches[3]
+	if len(port) != 0 {
+		return port
+	}
+	return ""
+}
```

Comment:
The provided code seems to be incomplete with certain sections missing, including the `LoadFromCert`, etc. However, based on what you've shared, there seem to be some regular coding standards violated like using strings directly instead of a struct for error handling and variable names that don't follow Go conventions.

Here's a basic outline:

### Check for Regular Irregularities

- Check for unnecessary or irrelevant comments.
- Ensure consistent use of single quotes around variables.
- Ensure function parameter naming follows proper convention (for example, without space 'Port' should ideally translate into 'Port').
- Make sure imports are organized correctly.
- Verify consistency between interface and implementation parts where applicable.